packages/ui/react-ui-editor/src/styles/layout.ts

```diff
@@ -2,11 +2,8 @@
 // Copyright 2024 DXOS.org
 //
 
+// TODO(burdon): Remove these styles which are specific to plugin usage, not component?
```

Comment:
These styles are here mainly because Toolbar is also defined here, and it was useful to offer fragments for laying the toolbar and editor in an expected configuration. They are also used in the stories.